Source/lighting.cpp

```diff
@@ -849,12 +849,6 @@ void MakeLightTable()
 		tbl += 240;
 	}
 
-	LoadFileInMem("PlrGFX\\Infra.TRN", tbl, 256);

```

Comment:
The previous approach seems better -- no allocations or string map lookups

Reply:
yes but it used the lighttable, the goal of this PR was to make TRNs usable by name and not only by some voodoo index

Reply:
Extract the voodoo index into a constant?

Reply:
Restored + added functions to get these TRNs